spec/codebreaker/game_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Codebreaker
+  RSpec.describe Game do
+    subject(:game) { described_class.new(Difficulty.find('easy').level) }
+
+    before do
+      stub_const('Codebreaker::Storage::STATS_DB', './lib/db/test-stats.yml')
```

Comment:
stub Storage in this tests, you have separate tests for it, you don't need to test it second time